src/App.js

```diff
@@ -15,10 +15,13 @@ class App extends React.Component {
     this.state = {
       listStudents: [],
       //listCourses: [],
-      //activeStudent:'',
+      activeStudent: [],
       //activeCourse:''
+      selectedStudent: [],
```

Comment:
the selectedStudent is never used, we should delete it

Reply:
yes, i didnt check it that good!